### PR TITLE
Introduction

### PR DESCRIPTION
**Pull request by pedrosousabarreto**
This is the initial version of the intro.

Next will be a merge with of the intro text on the Reference Architecture section, that should be move to the intro section.

### CODE REVIEW

File: docs/introduction/README.md [hunk 1]
```diff
@@ -1,11 +1,57 @@
...
+* Solution Space Context mapping - grouping similar problems together based on their purpose and context
+* Individual use case mapping - where all use cases where discussed and detailed in perspective with their objectives and the rest of the system
+
+### Problem Space mapping
+This exercise allows us to understand the different problem domains and subdomains, and how they are classified according to their importance.
+Considering that this initial version of the Reference Architecture focused exclusively on the Mojaloop Switch, we only have one domain. All internal problems are thus called problem subdomains.
+
+According to DDD we classified some problem subdomains as:
+* Core subdomains - we need to solve these problems ourselves to control the outcome. In practical terms, this is where we put the most effort. Examples are transfers, participant lifecycle management or keeping accounts and balances
+* Supporting subdomains - these are problems that can’t be easily solved by using off-the-shelf software, we’ll need some coding effort to solve them, it’s not just integration. Examples of this are Authorization and Access Policies or Reporting
```
Comment (mdebarros) [inline docs/introduction/README.md:46]:
```suggestion
* Supporting subdomains - This domain category provides ancillary functions that are required by the business domain. The problems within this category can generally be solved by integrating off-the-shelf software with some coding effort. Examples of this are Authorization and Access Policies or Reporting
```

File: docs/introduction/README.md [hunk 2]
```diff
@@ -1,11 +1,57 @@
...
+* Generic subdomains - these are usually problems that can be solved using of-the-shelf software, requiring only some integration effort. Examples are Authentication or Platform monitoring, we don’t need to code these solutions ourselves but we still to design accordingly
```
Comment (mdebarros) [inline docs/introduction/README.md:47]:
```suggestion
* Generic subdomains - Similar to the Supporting subdomain, except these problems are primarily solved by integrating off-the-shelf software that requires very little (if any at all) coding effort. Examples are Authentication or Platform monitoring, we don’t need to code these solutions ourselves but we still have to design accordingly
```

**Review by mdebarros — CHANGES_REQUESTED** [review]
Hey @pedrosousabarreto,

I have 3 major issues that need to be fixed:

1. Merge conflict;
2. Duplicate menu config; and
3. Missing notes

The rest are mostly suggestions on how the wording can be simplified, or changed to clarify the idea.

Have a look, and let me know your thoughts.

File: docs/introduction/README.md [hunk 3]
```diff
@@ -1,11 +1,57 @@
...
+Another important benefit is the guidance it provides to decision-making regarding technology choices and implementation strategies. 
+With the Reference Architecture in mind we can frame any development decision as tactical or strategic:
+- Tactical - something that is required right now and might ignore the reference architecture for the sake of an urgent value.
+- Strategic - something that is long-lasting and should be implemented in accordance to the Reference Architecture - should take the implementation closer to it.
+
+Last but not less important, it ensures alignment between the technical vision and the more important product vision (see below regarding the process and ways of working).   
+
+## Process of creating and maintaining the Reference Architecture
+
+The Reference Architecture creation process has been guided by Domain-Driven Design[1] principles and the solution space heavily inspired by SOLID's[2] single responsibility principle (SRP).
```
[the remark this answers is not in the document]
Comment (pedrosousabarreto) [reply docs/introduction/README.md:33]: This was copies from Michael's section.. the part we need to merge with later.

File: docs/introduction/README.md [hunk 4]
```diff
@@ -1,11 +1,57 @@
...
+* Problem Space mapping - what are the different problem domains and subdomains, and how they are classified according to their importance 
+* Solution Space Context mapping - grouping similar problems together based on their purpose and context
+* Individual use case mapping - where all use cases where discussed and detailed in perspective with their objectives and the rest of the system
+
+### Problem Space mapping
+This exercise allows us to understand the different problem domains and subdomains, and how they are classified according to their importance.
+Considering that this initial version of the Reference Architecture focused exclusively on the Mojaloop Switch, we only have one domain. All internal problems are thus called problem subdomains.
+
+According to DDD we classified some problem subdomains as:
+* Core subdomains - we need to solve these problems ourselves to control the outcome. In practical terms, this is where we put the most effort. Examples are transfers, participant lifecycle management or keeping accounts and balances
```
[the remark this answers is not in the document]
Comment (pedrosousabarreto) [reply docs/introduction/README.md:45]: This not accurate, but I'll come to this later.

File: docs/introduction/README.md (hunk 1 above)
Comment (pedrosousabarreto) [reply docs/introduction/README.md:46]: same

**Comment by pedrosousabarreto** [comment]
Thanks for the review, accepted most. Some require a bit more thinking (copied from slides).
This was just a draft submission just to push my changes while out-of-office

File: docs/.vuepress/config.js [hunk 5]
```diff
@@ -76,37 +76,19 @@ module.exports = {
...
     smoothScroll: true,
     logo: '/mojaloop_logo_med.png',
-    sidebarDepth: 2,
+    sidebarDepth: 1,
     lastUpdated: true,
     nav: [
       {
-        text: 'Getting Started',
-        link: '/gettingStarted/',
+        text: 'Introduction',
```
[the remark this answers is not in the document]
Comment (pedrosousabarreto) [reply docs/.vuepress/config.js:83]: I'm not following.. The first is the nav bar on top, the second is the sidebar.

**Comment by pedrosousabarreto** [comment]
Fixed the merge conflict in docs/introduction/README.md

**Review by mdebarros — APPROVED** [review]
+1

**Pull request merged**